app/src/main/res/layout/fragment_wallpaper_settings.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
License please. 📑

Reply:
📝

Reply:
Added. Is there a hook I could add to catch these?